### PR TITLE
Fixes #116: Added send button next to message box

### LINKED ISSUE #116: Add Send Button next to the message box.
**Actual Behaviour**

Currently users have to press enter to send the message,But it is bit confusing for new users.

**Expected Behaviour**

But if we can add send button  next to the message box it is little bit easy for new users.

### PR DESCRIPTION
**Pull request by madhavrathi**
Fixes issue #116

Changes: Added Button next to message box for sending message

Demo Link:  http://button.surge.sh/

Screenshots for the change: 
picc

### CODE REVIEW

**Comment by madhavrathi** [comment]
@rishiraj824  Please have a look http://button.surge.sh/

**Comment by madhavrathi** [comment]
picccc
How about this ?
i have updated by PR to look like this,

**Comment by madhavrathi** [comment]
final
Demo link: http://button.surge.sh/

@rishiraj824 @uday96 @isuruAb Please review.

File: src/components/ChatApp.css [hunk 1]
```diff
@@ -182,7 +182,7 @@ header{
     overflow-y: hidden;
 }
 .message-composer {
-  width: 100%;
+  width: 600px;
```
Comment (rishiraj824) [inline src/components/ChatApp.css]: This creates a scroll bar on the phone. The message composer is not responsive.

**Comment by madhavrathi** [comment]
@rishiraj824 @isuruAb  Made the required changes. 
demo: http://button-new.surge.sh/
Thanks for your reviews. Let me know if anything else is not working
Thanks

**Comment by rishiraj824** [comment]
@madhavrathi The font of the textarea has changed, please make it `Lato`.
The one in http://chat.susi.ai is Lato

**Comment by madhavrathi** [comment]
@rishiraj824 Fixed. Font is lato in text area

**Pull request merged**